security/target_path.rst

```diff
@@ -69,4 +38,42 @@ Next, create your own ``ExceptionListener``::
         }
     }
 
-Add as much or as little logic here as required for your scenario!
+By preventing ``setTargetPath()`` from being called on the parent, the Security component
+won't retain the request URI. Add as much or as little logic here as required for your scenario!
+
+Next, create the ``ExceptionListenerPass`` to replace the definition of the default
+``ExceptionListener`` with the one you just created. Make sure to use the name of 
+the firewall in your security configuration::
+
+    // src/AppBundle/DependencyInjection/Compiler/ExceptionListenerPass.php
+    namespace AppBundle\DependencyInjection\Compiler;
+    
+    use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+    use Symfony\Component\DependencyInjection\ContainerBuilder;
+    use AppBundle\Security\Firewall\ExceptionListener;
+
+    class ExceptionListenerPass implements CompilerPassInterface
+    {
+        public function process(ContainerBuilder $container)
+        {
+            // Use the name of your firewall for the suffix e.g. 'secured_area'
+            $definition = $container->getDefinition('security.exception_listener.secured_area');
+            $definition->setClass(ExceptionListener::class);
+        }
+    }
+
+Finally, add a compiler pass to tie it all together::
+
+    // src/AppBundle/AppBundle.php
+    namespace AppBundle;
+    
+    use AppBundle\DependencyInjection\Compiler\ExceptionListenerPass;
+
+    class AppBundle extends Bundle
```

Comment:
I mean using https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpKernel/Kernel.php#L446 instead, but this is not the good place to ask, and I guess will be part of a bigger update to get rid of AppBundle in the docs. So never mind :)